src/js/api/brand.js

```diff
@@ -142,14 +142,15 @@ ripe.Ripe.prototype.getFontUrl = function(name, format, options) {
 /**
  * Returns the URL for the environment file of the 3D scene.
  *
- * @param {Object} options A map with options, such as:
- *  - 'brand' - The brand of the model.
- *  - 'model' - The name of the model.
- *  - 'version' - The version of the build, defaults to latest.
+ * @param {String} name The name of the environment map.
+ * @param {String} format The file format of the environment map.
+ * @param {Object} options A map with options.
  * @returns {String} The URL of the environment for the 3D scene.
  */
-ripe.Ripe.prototype.get3dSceneEnvironmentUrl = function(options) {
-    return "https://www.dl.dropboxusercontent.com/s/o0v07nn5egjrjl5/studio2.hdr";
+ripe.Ripe.prototype.get3dSceneEnvironmentUrl = function(name, format, options) {
+    name = name || "studio2";
+    format = format || "hdr";

```

Comment:
why not
```suggestion
ripe.Ripe.prototype.get3dSceneEnvironmentUrl = function(name = "studio2", format = "hdr", options) {

```

Also, our default hdr should be called studio instead of studio2
I think we should go as far as renaming the file in ripe_static

Reply:
That's because it's the file given from 3D, I don't actually know what is the original name

Reply:
The reason I don't put them in the arguments, is because the arguments are required and not optional. It's a fallback not a default

Reply:
> It's a fallback not a default

Is there a difference in this case?

Reply:
Yes! 😅 

Here it suggests the arguments are required. If you set like you suggestion, the method is letting you use those arguments are optional.

Some context for this type of usages:
The reason this fallback exists is for compatibility or it wouldn't even exist and just throw errors.
For example, a build that doesn't have the environment setting set, with this, will still work after this change (`dummy` `cube` for example) while still respecting the interface.

An alternative I guess would be to remove this from here and set it before it's usage. As I'm typing this now I see that this last approach is more correct so I will do that change now 🤔